v1.1.3: fixed missing deps and improve test

easz opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

Commit:
add missing deps

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

Commit:
upgrade version

Commit:
try lower setuptools

easz commented on recipe/meta.yaml:
```diff
@@ -44,6 +44,7 @@ requirements:
     - xmltodict
     - imagecodecs
     - albumentations
+    - setuptools <59.6.0
```
- due to https://github.com/pytorch/pytorch/issues/69894
- remove this version constraint after pytorch 1.10.2

bw4sz reviewed (commented):
Thanks for the PR. I need to understand the status. Locally @ethanwhite was having trouble building. But I see tests pass on CI for linux.

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=442980&view=results

passes. What error is this PR trying to fix?

ethanwhite commented:
This PR primarily addresses the missing albumentations dependency.

Generally I think we're up and running otherwise - see additional discussion in https://github.com/weecology/DeepForest/issues/288 this morning. Builds are still finicky but there are ways to do it consistently. There was also some sort of lag somewhere in the pipeline which cleaned itself up overnight (conda stopped trying to install 0.2.x).

easz commented:
the CI passed before, because it run only `import deepforest` as the single test.
It would fail with running `from deepforest import main`...
therefore I added the missing dependency, defined some version constraint and added a "test" in meta.yml.

you can review if the version constraint makes sense..
